samples/net/wifi/apsta_mode/socs/esp32_procpu.conf

```diff
@@ -3,3 +3,12 @@ CONFIG_ESP32_WIFI_STA_AUTO_DHCPV4=y
 CONFIG_ESP32_WIFI_AP_STA_MODE=y
 CONFIG_WIFI_NM=y
 CONFIG_WIFI_NM_MAX_MANAGED_INTERFACES=2
+
+# apsta_mode configuration
```

Comment:
This makes the sample application work the same way as before which I suppose is okeish, but the configuration should not be hardcoded in the application that affects all users of this soc. At some point these settings should be removed from this global file and be placed to an overlay file instead so that users wanting these values can use them. FYI @sylvioalves